data/sqlite_database.py

```diff
@@ -63,13 +63,13 @@ def get_netcdf_files(self, timestamp: List[int], variable: List[str]) -> List[st
             variable = ?
             AND timestamp = ?;
         """
-        for ts in timestamp:
+        for ts in sorted(timestamp):
```

Comment:
Nice one.

Reply:
Side note:
Now that I look at this function, the way it's requesting data from the DB is actually terrible because it's executing one query per timestamp per variable (yes I was the idiot who wrote this 2 years ago). SQL support multiple values in a single WHERE statement so it's really just a matter of passing `sorted(timestamp).join(',')` into `self.c.execute(...)` I think to eliminate the outer loop.

No need to action this since we're not experiencing performance issues yet, but this file is a prime candidate for optimization.